### PR TITLE
[EMB-85] Quick Files front-end

### PR DESCRIPTION
**Pull request by jamescdavis**
## Purpose

Enable Quick Files

## Changes

Bunch of stuff for Quick Files front-end

## QA Notes

Does Quick Files work?

## Side Effects

?

## Ticket

https://openscience.atlassian.net/browse/EMB-85

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -97,6 +117,7 @@ services:
 ##        ./node_modules/.bin/bower install --allow-root --config.interactive=false &&
 ##        yarn start --host 0.0.0.0 --port 4202 --live-reload-port 41955
 
+# Use this for ember-osf linked development (with docker-sync):
```
Comment (jamescdavis) [inline docker-compose.override.yml]: Not sure where this came from. Should not be here.

File: website/settings/local-dist.py [hunk 2]
```diff
@@ -41,6 +41,11 @@
         'server': 'http://192.168.168.167:4202',
         'path': '/registries/'
     },
+    'ember_osf_web': {
+        'url': '/ember_osf_web/',
+        'server': 'http://localhost:4201',
```
Comment (jamescdavis) [inline website/settings/local-dist.py]: this should be 4200

File: website/settings/local-travis.py [hunk 3]
```diff
@@ -32,6 +32,11 @@
         'url': '/preprints/',
         'server': 'http://localhost:4201',
         'path': os.environ.get('HOME') + '/preprints/'
+    },
+    'ember_osf_web': {
+        'url': '/ember_osf_web/',
+        'server': 'http://localhost:4201',
```
Comment (jamescdavis) [inline website/settings/local-travis.py]: 4200

**Comment by jamescdavis** [comment]
Personal build: https://travis-ci.org/jamescdavis/osf.io/builds/336660743

File: website/quickfiles/views.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+import requests
+from flask import send_from_directory, Response, stream_with_context
+
+from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS
+
+quickfiles_dir = os.path.abspath(os.path.join(os.getcwd(), EXTERNAL_EMBER_APPS['quickfiles']['path']))
```
Comment (sloria) [inline website/quickfiles/views.py]: Minor: Better to use `settings.APP_PATH` here.

Certainly not a blocker, though.

**Pull request merged**